src/main/webapp/src/components/songInfo.js

```diff
@@ -163,17 +191,8 @@ class SongInfo extends React.Component {
             </div>
             <div class="song-entity-analysis">
               <Typography variant="h4">Entity Analysis</Typography>
-              <List>
-                {songInfo.entityAnalysis.map((wordInfo, index) => (
-                  <ListItem key={index + 1}>
-                    {wordInfo.word +
-                      " - " +
-                      wordInfo.salience +
-                      " - " +
-                      wordInfo.category}
-                  </ListItem>
-                ))}
-              </List>
+              <button onClick={loadChartAPI}>Get Entity Analysis results</button>
```

Comment:
Of course we don't want force user to push some button just to see the charts. One of the best solution is to use `ref`.

Reply:
Oh ok I thought we wanted a button. My bad.